example/JSON/VEVEVO.json

```diff
@@ -0,0 +1,20 @@
+{
+    "line": "2",
+    "input": "./example/C2HDM_Input.dat",
```

Comment:
```suggestion
    "input": "../example/C2HDM_Input.dat",
```

Reply:
All these changes depend on where you call the executable. I call them from the toplevel of the project and I assume you call them from the build directory?

Reply:
I call them the same way we also write in the `Readme`, so from `build`. I would be consistent with the syntax.